libsrc/leddevice/dev_net/LedDeviceYeelight.cpp

```diff
@@ -0,0 +1,1503 @@
+﻿#include "LedDeviceYeelight.h"
+
+// ssdp discover
+#include <ssdp/SSDPDiscover.h>
+
+// Qt includes
+#include <QEventLoop>
+
+#include <QtNetwork>
+#include <QTcpServer>
+#include <QColor>
+#include <QDateTime>
+
+#include <chrono>
+#include <thread>
+
+static const bool verbose  = false;
+static const bool verbose3  = false;
+
+// Constants
+const int WRITE_TIMEOUT	= 1000;			 // device write timeout in ms
```

Comment:
What about this for unit type safety ?

    constexpr std::chrono::milliseconds WRITE_TIMEOUT{1000};

Then you can use ````WRITE_TIMEOUT.count()```` to get the value in msecs.

Reply:
Done.